src/cocoa/toga_cocoa/libs/appkit.py

```diff
@@ -7,7 +7,7 @@
 
 from rubicon.objc import *
 from toga.constants import *
-from toga.color import *
+from toga.resources.colors import *
 
 ######################################################################
 appkit = cdll.LoadLibrary(util.find_library('AppKit'))
```

Comment:
At column 10: (F405) 'cdll' may be undefined, or defined from star imports: ctypes, rubicon.objc, toga.constants, toga.resources.colors